src/app/label/[imageKey]/finish/page.tsx

```diff
@@ -50,6 +51,17 @@ const FinishPage = () => {
     );
   }
 
+  const handleCopyToClipboard = async () => {
+    try {
+      const shareUrl = `${process.env.NEXT_PUBLIC_BASE_URL}?${ENTRY_QUERY.KEY}=${ENTRY_QUERY.VALUE}`;
+      await navigator.clipboard.writeText(shareUrl);
+      toast.success('링크 복사 성공');
+    } catch (error) {
+      toast.error('링크 복사 실패');
```

Comment:
사용자에게 보이는 부분인 만큼 조금 부드러운 문구면 좋을 것 같아요!
```suggestion
      toast.success('링크를 복사했어요');
    } catch (error) {
      toast.error('링크 복사에 문제가 발생했어요. 다시 시도해주세요');
```